draft/implementation-notes/index.md

```diff
@@ -672,7 +677,7 @@ checksums - considering some sort of failure has just occurred.
 <span id="ref-oais"/>**\[OAIS]** Reference Model for an Open Archival Information System (OAIS), Issue 2. June 2012.
 URL: <https://public.ccsds.org/pubs/650x0m2.pdf>
 
-<span id="ref-ocfl-specification"/>**\[OCFL-Specification]** OCFL Specification. URL: <https://ocfl.io/draft/spec>
+<span id="ref-ocfl-specification"/>**\[OCFL-Specification]** OCFL Specification v1.1. URL: <https://ocfl.io/1.1/spec>
```

Comment:
Should we instead leave this as `draft`?

Reply:
Yes, fixed in [360e855](https://github.com/OCFL/spec/pull/615/commits/360e8556a4e75a0eaa7efb62641a159563c95318)